lacework/resource_lacework_resource_group_aws.go

```diff
@@ -0,0 +1,192 @@
+package lacework
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"log"
+	"strings"
+
+	"github.com/lacework/go-sdk/api"
+)
+
+func resourceLaceworkResourceGroupAws() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceLaceworkResourceGroupAwsCreate,
+		Read:   resourceLaceworkResourceGroupAwsRead,
+		Update: resourceLaceworkResourceGroupAwsUpdate,
+		Delete: resourceLaceworkResourceGroupAwsDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: importLaceworkIntegration,
```

Comment:
we need to implement a new importer for Resource Groups.
```suggestion
			State: importLaceworkResourceGroup,
```